lib/mrkt/concerns/crud_custom_activities.rb

```diff
@@ -0,0 +1,39 @@
+require 'time'
+
+module Mrkt
+  module CrudCustomActivities
+    def get_list_of_custom_activity_types()
+      get('/rest/v1/activities/types.json')
+    end
+
+    def create_custom_activity(lead_id, activity_type_id, primary_attribute_value, attributes: {}, date: nil)
+      date ||= Time.now()
```

Comment:
Please do not add parens to the method calls either when there are no arguments.